pkg/sql/sqlbase/table.go

```diff
@@ -39,17 +39,17 @@ import (
 )
 
 var nameToVisibleTypeMap = map[string]ColumnType_VisibleType{
-	"INTEGER":         ColumnType_INTEGER,
-	"INT4":            ColumnType_INTEGER,
-	"INT8":            ColumnType_BIGINT,
-	"INT64":           ColumnType_BIGINT,
-	"BIT":             ColumnType_BIT,
-	"INT2":            ColumnType_SMALLINT,
-	"SMALLINT":        ColumnType_SMALLINT,
-	"FLOAT4":          ColumnType_REAL,
-	"REAL":            ColumnType_REAL,
-	"FLOAT8":          ColumnType_DOUBLE_PRECISON,
-	"DOUBLE PRECISON": ColumnType_DOUBLE_PRECISON,
+	"INTEGER":          ColumnType_INTEGER,
+	"INT4":             ColumnType_INTEGER,
+	"INT8":             ColumnType_BIGINT,
+	"INT64":            ColumnType_BIGINT,
+	"BIT":              ColumnType_BIT,
+	"INT2":             ColumnType_SMALLINT,
+	"SMALLINT":         ColumnType_SMALLINT,
+	"FLOAT4":           ColumnType_REAL,
+	"REAL":             ColumnType_REAL,
+	"FLOAT8":           ColumnType_DOUBLE_PRECISION,
+	"DOUBLE PRECISION": ColumnType_DOUBLE_PRECISION,
```

Comment:
Fairly sure this is fixing a real but minor bug - I think without this change users wouldn't be able to specify a DOUBLE PRECISION type with a precision type omdifier. Or something like that.

Reply:
Looks like there's a test failure so you might be right. I'll update the release notes if this is the case.

Reply:
Yep, was a minor bug, added the release note.